benchmark/collective/resnet/utils/reader_cv2.py

```diff
@@ -32,11 +33,15 @@ def _reader_creator(settings,
                     pass_id_as_seed=0,
                     threads=4,
                     buf_size=4000):
-    def reader():
+    def _reader():
         with open(file_list) as flist:
             full_lines = [line.strip() for line in flist]
-            if shuffle:
-                random.Random(pass_id_as_seed).shuffle(full_lines)
+            if shuffle and pass_id_as_seed is not None:
+                if (not hasattr(_reader, 'seed')):
+                    _reader.seed = pass_id_as_seed
+                random.Random(_reader.seed).shuffle(full_lines)
+                print("reader.seed", _reader.seed)
```

Comment:
clean up this?

Reply:
reserved for debugging?
![image](https://user-images.githubusercontent.com/10208305/70206546-d1d45500-171f-11ea-9712-16be286d017f.png)